spec/models/health_care_application_spec.rb

```diff
@@ -189,9 +188,15 @@
     end
 
     context 'with a loa1 user' do
-      it 'returns partial ee data' do
+      it 'returns partial ee data when enrollment_status is present' do
         expect(described_class.parsed_ee_data(ee_data, false)).to eq(
-          parsed_status: login_required
+          parsed_status: HCA::EnrollmentEligibility::Constants::LOGIN_REQUIRED
+        )
+      end
+
+      it 'returns none of the above ee data when enrollment_status is not set' do
+        expect(described_class.parsed_ee_data({}, false)).to eq(
+          parsed_status: HCA::EnrollmentEligibility::Constants::NONE_OF_THE_ABOVE
```

Comment:
Could refactor these to examples into this, just to increase consistency with the spec a bit

when enrollment_status is present

```ruby
context 'when enrollment_status is present' do 
  it 'returns partial ee data' .... 
end

context 'when enrollment_status is not set' do 
  it 'returns none of the above ee data' ..
end
```

Reply:
Yeah, I agree that makes it read more clearly. I'll update it 👍